torch/utils/mkldnn.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+
+
+def to_mkldnn(module):
+    for child in module.children():
+        to_mkldnn(child)
+
+    def fn(t):
+        if t.is_floating_point():
+            return t.to_mkldnn()
+
+    for param in module._parameters.values():
```

Comment:
it assumes that all ops work on mkldnn which might not be the case. Maybe we should have two versions: conservative that matches only some modules and full-on that handles all params

Reply:
Hmm supporting partial conversion is hard I think, you will need to add `to_mkldnn` and `to_dense` at the boundaries.